integrations/common/marquez/dataset.py

```diff
@@ -35,7 +35,7 @@ def __init__(
             raise RuntimeError('scheme + authority and namespace are exclusive options')
 
     def __eq__(self, other):
-        return self.name == other.name and \
+        return self.name == other.job_name and \
```

Comment:
Was this rename intentional? It's seems that we are now comparing the dataset name with the job name?

Reply:
Fixed.